switch statement

hsy00 opened the pull request:
adding another switch statements

hsy00 committed:
switch statement

AkanshDivker reviewed (changes requested):
Convert this to a class like we discussed in our last meeting, and then everything should be good to go!

hsy00 committed:
Delete switch_junk.c

hsy00 committed:
Delete insert_switch.py

hsy00 committed:
Update switch statement

1. Adding random variables
2. Generating new codes inside the python file
3. Minor change for processing.py
